### ISSUE TITLE
More detailed permissions for data output.

### ISSUE THREAD

**Issue by r15ch13** [Valandur/Web-API#10]
For my server, I would like to make parts of the API public. Mainly because I recently created a player position modification for [overviewer](https://overviewer.org/) maps.
So it would be cool to have a more detailed permission system to restrict access to certain parts of data.
For example, I don't need the `inventory`, `connection`, `achievements` and most of the `statistics`

Example permissions for my use case:
```
player.name
player.location.*
player.armor
player.health
player.experience
player.gameMode
player.food
player.statistics.deaths
```

Also restricting the access to POST endpoints would be nice.

**Comment by Valandur** [comment]
This has been partly implemented in 190f527b28d5c90bf023937055696cb05349e48e and is in releases starting with v3.2.0.

The restricting of actual data has not yet been implemented, this will take a while as it's more involved, I am however already working on it.

**Comment by Valandur** [comment]
This is implemented in 284ae5268677d913d32071b8be5814cb07368db0, starting with version [3.3.0](https://github.com/Valandur/Web-API/releases/tag/v3.3.0).  
Documentation can be found [here](https://github.com/Valandur/Web-API/blob/master/docs/PERMISSIONS.md) and more generally [here](https://github.com/Valandur/Web-API/blob/master/docs/INDEX.md)

**Comment by r15ch13** [comment]
Looks awesome 👍
